api/nodes/views.py

```diff
@@ -152,6 +159,48 @@ def get_queryset(self):
         return self.get_queryset_from_request()
 
 
+class NodeContributorDetail(generics.RetrieveUpdateDestroyAPIView, NodeMixin):
+    """Detail of a contributor for a node.
+
+    Contributors are users who can make changes to the node or, in the case of private nodes,
+    have read access to the node. Contributors are divided between 'bibliographic' and 'non-bibliographic'
```

Comment:
I'm not sure this info is necessary; feels like too much detail for the purpose of API endpoint documentation. Besides, the semantics of "bibliographic" and "non-bibliographic" should be self-explanatory.